src/test/skript/tests/syntaxes/expressions/ExprDuplicateCooldown.sk

```diff
@@ -0,0 +1,32 @@
+# includes: ExprDuplicateCooldown, EffAllayDuplicate, EffAllayCanDuplicate, CondAllayCanDuplicate
+test "duplicate cooldown":
+	spawn an allay at test-location:
+		set {_entity} to entity
+
+	set the duplication cooldown of {_entity} to 5 seconds
+	assert the duplication cooldown of {_entity} is 5 seconds with "Duplication cooldown was not set"
+	add 3 seconds to the duplication cooldown of {_entity}
+	assert the duplication cooldown of {_entity} is 8 seconds with "Duplication cooldown did not have 3 seconds added"
+	remove 5 seconds from the duplication cooldown of {_entity}
+	assert the duplication cooldown of {_entity} is 3 seconds with "Duplication cooldown did not have 5 seconds removed"
+	clear the duplication cooldown of {_entity}
+	assert the duplication cooldown of {_entity} is 0 seconds with "Duplication cooldown time was not cleared"
+	set the duplication cooldown of {_entity} to 1 hour
+	remove 2 hours from the duplication cooldown of {_entity}
+	assert the duplication cooldown of {_entity} is 0 seconds with "Duplication cooldown should be 0 seconds after removing more than current"
+	add (2 seconds * -1) to the duplication cooldown of {_entity}
+	assert the duplication cooldown of {_entity} is 0 seconds with "Duplication cooldown should be 0 seconds after adding a negative"
+	set the duplication cooldown of {_entity} to (1 minute * -1)
```

Comment:
I forgot that this accepts timespans, so these can be removed. Timespans will not go below 0, 1 minute * -1 is 0 minutes, so these aren't actually testing anything.